types/node/node-tests.ts

```diff
@@ -3198,8 +3198,9 @@ namespace http2_tests {
             let headersSent: boolean = response.headersSent;
 
             response.setTimeout(0, () => {});
-            response.createPushResponse(outgoingHeaders);
-            response.createPushResponse(outgoingHeaders, (err: Error) => {});
+            response.createPushResponse(outgoingHeaders, () => {});
```

Comment:
Don't need all these tests, it's a TypeScript feature that you can ignore as many parameters as you want; so just include the last test.

Reply:
Sounds good. I was just following the pattern in other lines in this file. Done.